tests/auxtel/test_build_pointing_model.py

```diff
@@ -22,13 +22,30 @@
 import unittest
 import contextlib
 import unittest.mock
+import pathlib
+import tempfile
 
 import numpy as np
 
 from lsst.ts.standardscripts import BaseScriptTestCase
 from lsst.ts.externalscripts import get_scripts_dir
 from lsst.ts.externalscripts.auxtel.build_pointing_model import BuildPointingModel
 from lsst.ts.observatory.control.utils.enums import RotType
+import lsst.daf.butler as dafButler
+
+# Declare the local path that has the information to build a
+# local gen3 butler database
+
+DATAPATH = (tempfile.TemporaryDirectory(prefix="butler-repo")).name
+butler_config_path = (
```

Comment:
is there a better way to do this pathlib stuff? It's not pretty

Reply:
Implemented as discussed.

Reply:
I don't see the changes pushed, but glad the `getPackageDir` thing sounds like it worked.